client/src/components/pages/Home/style.css

```
@@ -0,0 +1,63 @@
+.home{
+    margin-top:0px;
+}
+.home__header{
+    margin: 50px 20px 5px 20px;
+    display: inline-flex;
+    width: 90%;
+}
+.home__box{
+    margin-top: 52px;
+}
+.home__title{
+    font-size: 36px;
+    font-weight: bold;
+    font-family: Roboto;
```

This will be better if the URL font is had any problem.
```suggestion
    font-family: Roboto, sans-serif;
```